plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/Layer.java

```diff
@@ -231,39 +231,6 @@
      */
     public void setInteraction(Interaction interaction, boolean isApplicable);
 
-    /**
-     * Indicates this layer is capable of selectable.
-     * <p>
-     * A Selectable Layer can be used with the Utilities.getQuery opperation. The selection tool
-     * category will maintain a separate user interface concept of applicability. The selection tool
-     * will not be capabile of considering a non selectable layer applicable.
-     * </p>
-     * IMPORTANT FUTURE CHANGE NOTE: future design: Instead of isSelectable and isInfoable a single
-     * boolean isApplicable( String toolkitID ) will be used to determine whether the layer is
-     * applicable to the current tool cateogry. If the layer is not capable of Each tool category
-     * extension will declare a color, which will be an underlay for the layer decorator, and an
-     * optional validator class, to determine whether the capability for the layer can be set.
-     * Indicates this layer is selectable.
-     *
-     * @return <code>true</code> if layer is selectable, <code>false</code> otherwise.
-     * @deprecated use getInteraction(Interaction.SELECT)
-     */
-    @Deprecated
-    public boolean isSelectable();
-
-    /**
-     * Sets the value of the '{@link org.locationtech.udig.project.internal.Layer#isSelectable
-     * <em>Selectable</em>}' attribute. <!-- begin-user-doc --> Used by the user to control which
-     * layers are selectable, may be ignored for GeoResources that do not support editing. <!--
-     * end-user-doc -->
-     *
-     * @param value the new value of the '<em>Selectable</em>' attribute.
-     * @see #isSelectable()
-     * @deprecated use setInteraction(Interaction.SELECT, value)
-     */
-    @Deprecated
-    void setSelectable(boolean value);
-
```

Comment:
these interface methods are generated from the model. I guess its necessary to remove these from EMF model and re-generate code usin EMF Toolset within Eclipse.

Maybe @egouge can support here?